src/webview-js/cnxml-preview/cnxml-preview.js

```diff
@@ -128,6 +128,7 @@ function sendUpdatedXML() {
 //
 // From https://github.com/philschatz/accessible-engine/blob/master/src/browser/vdom.ts
 // Original: https://medium.com/@deathmood/how-to-write-your-own-virtual-dom-ee74acc13060
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Could this be switched to use Preact without too much additional work? I included a homegrown vdom for spike reasons but this PR adds Preact and webpack for Webviews

Reply:
I can make a card for it, but I'd prefer not to include it in this work.

Reply:
card created